roxctl/image/scan/scan_test.go

```diff
@@ -260,10 +260,11 @@ func (s *imageScanTestSuite) newTestMockEnvironmentWithConn(conn *grpc.ClientCon
 
 func (s *imageScanTestSuite) SetupTest() {
 	s.defaultImageScanCommand = imageScanCommand{
-		image:      "nginx:test",
-		retryDelay: 3,
-		retryCount: 3,
-		timeout:    1 * time.Minute,
+		image:        "nginx:test",
+		retryDelay:   3,
+		retryCount:   3,
+		timeout:      1 * time.Minute,
+		retryTimeout: 1 * time.Minute,
```

Comment:
Don't we want to introduce a RetryPolicy environment object, which would incorporate all those properties as well as the `retryTimeout`?

Reply:
Added `retryTimeout` as an option to the gRPC connection. `retryDelay` / `retryCount` are specific to `roxctl/image/scan` and don't align with the retry behavior of other commands (which use retry on gRPC level). So I think a refactor would be necessary to generalize these parameters, which is out of scope for this PR.